source/dem/dem.cc

```diff
@@ -829,6 +1044,9 @@ DEMSolver<dim>::solve()
                                           triangulation,
                                           triangulation_cell_diameter);
 
+  // Setup background dof
+  setup_background_dofs();
```

Comment:
I am not sure why you need  dofs here?

Reply:
I use it for the identify mobility status and for the and output